components/list/list-item-checkbox-mixin.js

```diff
@@ -140,6 +130,10 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(L
 		}
 	}
 
+	_onNestedSlotChange() {
```

Comment:
This addresses an issue that can arise when the last selectable item is removed from a nested list.  Without this, selection state gets completely messed up.  This took a long time to figure out.  😬